server/candlepin.spec.tmpl

```diff
@@ -43,8 +43,7 @@ Requires: wget
 Requires: tomcat >= 0:7.0.0
 Requires: liquibase >= 0:3.0.0
```

Comment:
Please remove the `Requires: liquibase >= 0:3.0.0` line.

Reply:
Done.